rotkehlchen/db/schema.py

```diff
@@ -599,6 +599,23 @@
 );
 """  # noqa: E501
 
+DB_CREATE_HISTORY_EVENTS = """
+CREATE TABLE IF NOT EXISTS history_events (
+    identifier TEXT NOT NULL PRIMARY KEY,
+    event_identifier TEXT NOT NULL,
+    sequence_index INTEGER NOT NULL,
+    timestamp INTEGER NOT NULL,
+    location TEXT NOT NULL,
+    location_label TEXT,
+    asset TEXT NOT NULL,
+    amount TEXT NOT NULL,
+    usd_value TEXT NOT NULL,
```

Comment:
This table needs to also be added to the latest (v30_v31?) DB upgrade!

Reply:
As it doesn't exists, wouldn't it be generated at the moment of initializing the database?